tests/e2e/dynamic_provisioning_test.go

```diff
@@ -158,6 +158,7 @@ var _ = Describe("[fsx-csi-e2e] Dynamic Provisioning with s3 data repository", f
 						Parameters: map[string]string{
 							"subnetId":         subnetId,
 							"securityGroupIds": strings.Join(securityGroupIds, ","),
+							"autoImportPolicy": "NONE",
```

Comment:
is it needed, shouldnt aws default to NONE?

Reply:
That's correct that it's not needed and I verified the tests will still pass if it's not included. I included it just to be explicit here but I can remove the parameter if you think it doesn't add value.

Reply:
thanks no worries, just checking if it was related to test passing :+1: